yorkie/src/test/kotlin/dev/yorkie/document/crdt/CrdtArrayTest.kt

```diff
@@ -0,0 +1,132 @@
+package dev.yorkie.document.crdt
+
+import dev.yorkie.document.time.ActorID
+import dev.yorkie.document.time.TimeTicket
+import junit.framework.TestCase.assertEquals
```

Comment:
`junit.framework.TestCase` is deprecated.
We should use `assert*` functions from `org.junit.Assert` instead.